packages/plugin/src/theia-proposed.d.ts

```diff
@@ -569,6 +569,86 @@ export module '@theia/plugin' {
     }
 
     // #endregion
+    export interface WebviewView {
```

Comment:
@msujew should we move `WebviewView` and `WebviewViewProvider` instead under `theia.d.ts`? From what I can tell the API was finalized in vscode (under `vscode.d.ts`) even at `tag:1.53.2`,

Reply:
Good catch, yes I moved it to `theia.d.ts`.